composer.json

```diff
@@ -16,15 +16,15 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "^1.0@dev",
-        "php-http/discovery": "^0.4",
+        "php-http/httplug": "^1.0-beta",
+        "php-http/discovery": "^0.5",
```

Comment:
@sagikazarmark This gets somewhat tiresome. Would `~0.5` (so including `0.6` and up) be okay with you?